src/test/java/com/wikia/webdriver/common/core/helpers/Emulator.java

```diff
@@ -1,13 +1,12 @@
 package com.wikia.webdriver.common.core.helpers;
 
-import java.util.Map;
-
+import com.google.common.collect.ImmutableMap;
 import lombok.Getter;
 
-import com.google.common.collect.ImmutableMap;
+import java.util.Map;
 
 public enum Emulator {
-  GOOGLE_NEXUS_5(
+  GOOGLE_NEXUS_5("GOOGLE_NEXUS_5",
       new ImmutableMap.Builder<String, Object>()
           .put("width", 360)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "width" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)